bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "balance-text",
+  "version": "1.5.0",
+  "authors": [
+    "Adobe Systems",
+    "Inc. <https://github.com/adobe-webplatform>"
```

Comment:
@rileyjshaw I don't know anything about bower, but should "Adobe Systems" and "Inc. <https://github.com/adobe-webplatform>" really be separate entries for authors?